### PR TITLE
Remove code which was causing proxy crash

### PR DESCRIPTION
**Pull request by tomduncalf**
Crash could be reproduced with e.g.:
```
people = realm.objects(Person);
peopleProxy = new Proxy(people, {get: () => {} });
Object.prototype.toString.call(peopleProxy);
```

## What, How & Why?

This closes # ???

## ☑️ ToDos

* [x] 📝 Changelog entry
* [ ] 📝 `Compatibility` label is updated or copied from previous entry
* [ ] 🚦 Tests
* [ ] 📱 Check the React Native/other sample apps work if necessary
* [ ] 📝 Public documentation PR created or is not necessary
* [ ] 💥 `Breaking` label has been applied or is not necessary

*If this PR adds or changes public API's:*
* [ ] typescript definitions file is updated
* [ ] jsdoc files updated
* [ ] Chrome debug API is updated if API is available on React Native

### CODE REVIEW

File: src/node/node_class.hpp [hunk 1]
```diff
@@ -934,8 +934,8 @@ Napi::Value WrappedObject<ClassType>::ProxyHandler::get_own_property_descriptor_
     Napi::EscapableHandleScope scope(env);
 
     // Napi::Object target = info[0].As<Napi::Object>();
-    Napi::String key = info[1].As<Napi::String>();
-    std::string text = key;
+    // Napi::String key = info[1].As<Napi::String>();
```
Comment (kneth) [inline src/node/node_class.hpp:937]: Why not remove this, previous and next line?

Comment (tomduncalf) [reply src/node/node_class.hpp:937]: Yeah I guess I can remove them all! I wondered if the existing one was commented out for a reason

**Review by takameyer — APPROVED** [review]
Great job on finding this one!

**Comment by somebody32** [comment]
I wonder if this could help with another Proxy problem and Hermes? https://github.com/realm/realm-js/issues/4119

**Comment by kraenhansen** [comment]
@somebody32 unfortunately not. This PR applies only to our Node.js integration.

**Pull request merged**